bundle/config/mutator/paths/app_paths_visitor_test.go

```diff
@@ -0,0 +1,42 @@
+package paths
+
+import (
+	"testing"
+
+	"github.com/databricks/cli/bundle/config"
+	"github.com/databricks/cli/bundle/config/resources"
+	"github.com/databricks/cli/libs/dyn"
+	assert "github.com/databricks/cli/libs/dyn/dynassert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAppPathsVisitor(t *testing.T) {
+	root := config.Root{
+		Resources: config.Resources{
+			Apps: map[string]*resources.App{
+				"app0": {
+					SourceCodePath: "foo",
+				},
+			},
+		},
+	}
+
+	actual := visitAppPaths(t, root)
+	expected := []dyn.Path{
+		dyn.MustPathFromString("resources.apps.app0.source_code_path"),
+	}
+
+	assert.ElementsMatch(t, expected, actual)
+}
+
+func visitAppPaths(t *testing.T, root config.Root) []dyn.Path {
```

Comment:
Can you dedup these helpers? They're all the same except for the `VisitXyzPaths` function.

Reply:
Makes sense, I will merge to keep momentum and address as a follow-up